diff --git a/node_modules/@algolia/autocomplete-js/dist/esm/autocomplete.js b/node_modules/@algolia/autocomplete-js/dist/esm/autocomplete.js
index 0dcb8b1..8f050f4 100644
--- a/node_modules/@algolia/autocomplete-js/dist/esm/autocomplete.js
+++ b/node_modules/@algolia/autocomplete-js/dist/esm/autocomplete.js
@@ -61,7 +61,7 @@ export function autocomplete(_ref2) {
   var label = document.createElement('label');
   var resetButton = document.createElement('button');
   var dropdown = document.createElement('div');
-  var autocomplete = createAutocomplete(_objectSpread({
+  var autocomplete = createAutocomplete(_objectSpread({}, props,{
     onStateChange: function onStateChange(options) {
       var state = options.state;
       render(state);
@@ -70,7 +70,7 @@ export function autocomplete(_ref2) {
         props.onStateChange(options);
       }
     }
-  }, props));
+  }));
   var onResize = debounce(function () {
     if (!dropdown.hasAttribute('hidden')) {
       setDropdownPosition();
